prowler/providers/aws/services/organizations/organizations_service.py

```diff
@@ -0,0 +1,231 @@
+import json
+from typing import Optional
+
+from botocore.client import ClientError
+from pydantic import BaseModel
+
+from prowler.lib.logger import logger
+from prowler.lib.scan_filters.scan_filters import is_resource_filtered
+from prowler.providers.aws.aws_provider import generate_regional_clients
+
+
+################## Organizations
+class Organizations:
+    def __init__(self, audit_info):
+        self.service = "organizations"
+        self.session = audit_info.audit_session
+        self.audited_account = audit_info.audited_account
+        self.audit_resources = audit_info.audit_resources
+        global_client = generate_regional_clients(
+            self.service, audit_info, global_service=True
+        )
+        self.client = list(global_client.values())[0]
+        self.region = self.client.region
+        self.organizations = []
+        self.policies = []
+        self.delegated_administrators = []
+        self.__describe_organization__()
+
+    def __describe_organization__(self):
+        logger.info("Organizations - Describe Organization...")
+
+        try:
+            # Check if Organizations is in-use
+            try:
+                organization_desc = self.client.describe_organization()["Organization"]
+                organization_arn = organization_desc["Arn"]
+                organization_id = organization_desc["Id"]
+                organization_master_id = organization_desc["MasterAccountId"]
+                organization_available_policy_types = organization_desc[
+                    "AvailablePolicyTypes"
+                ]
+                # Fetch policies for organization:
+                organization_policies = self.__list_policies__(
+                    organization_available_policy_types
+                )
+                # Fetch delegated administrators for organization:
+                organization_d_administrator = self.__list_delegated_administrators__()
```

Comment:
Could you change `organization_d_administrator` for `organization_delegated_administrator`?

Reply:
🚀